src/context/base.rs

```diff
@@ -21,6 +21,7 @@ use crate::{node, AudioListener};
 #[allow(clippy::module_name_repetitions)]
 pub trait BaseAudioContext {
     /// Returns the [`BaseAudioContext`] concrete type associated with this `AudioContext`
+    #[doc(hidden)] // we'd rather not expose the ConcreteBaseAudioContext
```

Comment:
Why not `pub(crate)` then?

Reply:
I believe a pub trait must have all items pub. Will check later